test/test_jit_cuda_fuser.py

```diff
@@ -1640,6 +1657,73 @@ def test2(x: torch.Tensor, y: torch.Tensor):
         )[0].graph
         FileCheck().check("aten::mul_").run(bwd2_graph)
 
+    @unittest.skipIf(not RUN_CUDA, "requires CUDA")
+    @unittest.skipIf(GRAPH_EXECUTOR != ProfilingMode.PROFILING,
+                     "Requires fusion optimization pass to be effective")
+    def test_dropout_inference_fusion(self):
+        dtype = torch.float
+        device = "cuda"
+        x = torch.randn([10, 4, 8], dtype=dtype, device=device)
+
+        def t(x: torch.Tensor, p: float, train: bool):
+            o = torch.nn.functional.dropout(x, p, training=train)
+            o = o + 1.0
+            return o
+
+        t_jit = torch.jit.script(t)
+
+        self._run_helper(t_jit, t, x, 0.15, False)
+
+    @unittest.skipIf(not RUN_CUDA, "requires CUDA")
+    @unittest.skipIf(GRAPH_EXECUTOR != ProfilingMode.PROFILING,
+                     "Requires fusion optimization pass to be effective")
+    def test_dropout_training_fusion(self):
+        dtype = torch.float
+        device = "cuda"
+        x = torch.randn([10, 4, 8], dtype=dtype, device=device, requires_grad=True)
+        grads = torch.randn([10, 4, 8], dtype=dtype, device=device)
+
+        def t(x: torch.Tensor, p: float, train: bool):
+            o = torch.nn.functional.dropout(x, p, training=train)
+            o = o + 1.0
+            return o
+
+        t_jit = torch.jit.script(t)
+
+        # The drop probability needs to be set to zero given that the order of picking random
```

Comment:
I thought for small tensor, our launch config should be giving identical RNG numbers (if we disable loop switching).